src/state/content-settings/__tests__/actions.test.js

```diff
@@ -185,4 +235,103 @@ describe('contentSettings thunks', () => {
       done();
     }).catch(done.fail);
   });
+
+  it('sendPostMetadataMap', (done) => {
+    store.dispatch(sendPostMetadataMap('key', 1)).then(() => {
+      expect(postMetadataMap).toHaveBeenCalledWith({ key: 'key', mapping: 1 });
+      const actions = store.getActions();
+      expect(actions[0]).toHaveProperty('type', TOGGLE_LOADING);
+      expect(actions[1]).toHaveProperty('type', SET_METADATA_MAPPING);
+      expect(actions[2]).toHaveProperty('type', TOGGLE_LOADING);
+      expect(actions[2].payload.id).toEqual('postMetadataMap');
+      done();
+    }).catch(done.fail);
+  });
+
+  it('sendPostMetadataMap error', (done) => {
+    postMetadataMap.mockImplementationOnce(mockApi({ errors: true }));
+    store.dispatch(sendPostMetadataMap('key', 2)).then(() => {
+      expect(postMetadataMap).toHaveBeenCalledWith({ key: 'key', mapping: 2 });
+      const actions = store.getActions();
+      expect(actions[0]).toHaveProperty('type', TOGGLE_LOADING);
+      expect(actions[0].payload.id).toEqual('postMetadataMap');
+      expect(actions[1]).toHaveProperty('type', 'errors/add-errors');
+      expect(actions[2]).toHaveProperty('type', TOGGLE_LOADING);
+      done();
+    }).catch(done.fail);
+  });
+
+  it('sendPutMetadataMap', (done) => {
+    const key = 'key';
+    const mapping = 1;
+    store.dispatch(sendPutMetadataMap(key, mapping)).then(() => {
+      expect(putMetadataMap).toHaveBeenCalledWith(key, mapping);
+      const actions = store.getActions();
```

Comment:
Same with [this](https://github.com/entando/entando-cms/pull/24/files#r334602809).
```suggestion
      const actions = store.getActions();
      expect(actions).toHaveLength(3);
```